program/src/instructions/init_update_mint_proof_v2.rs

```diff
@@ -26,14 +26,6 @@ pub struct InitUpdateMintProofV2<'info> {
     /// The whitelist account that the mint proof must validate against.
     // No constraints checks here because creating the mint proof is permissionless.
     // As long as the mint proof validates against the whitelist, it's valid.
-    #[account(
-        seeds = [
-            WhitelistV2::PREFIX,
-            &whitelist.namespace.key().as_ref(),
-            &whitelist.uuid
-        ],
-        bump = whitelist.bump,
-    )]
```

Comment:
Why not keep for redundancy

Reply:
What check does it actually do here that isn't covered by the Anchor account check on `Whitelist` type?